templates/global/wrapper-start.php

```diff
@@ -41,10 +44,7 @@
 	case 'twentysixteen' :
 		echo '<div id="primary" class="content-area twentysixteen"><main id="main" class="site-main" role="main">';
 		break;
-	case 'underscores' :
-		echo '<div id="primary" class="content-area"><main id="main" class="site-main" role="main">';
-		break;
 	default :
-		echo '<div id="container"><div id="content" role="main">';
+		echo '<div id="primary" class="content-area"><main id="main" class="site-main" role="main">';
```

Comment:
This break backwards compatibility for every non-underscores based theme currently using #container or #content.